hardware-computing/README.md

```diff
@@ -1,11 +1,95 @@
 # Secure Hardware Computing
 
```

Comment:
Add a list of "Prerequisites", as described in the [template-public](https://github.com/security-summer-school/template-public/tree/master/session-name-1#reminders-and-prerequisites) repo. Use it to specify that students should already be familiar with the concepts and encryption algorithms (AES, TCB, TPM).